tests/test_minimal/test_tools/test_nwb_helpers.py

```diff
@@ -25,12 +26,10 @@ def test_get_module(self):
         assert mod_2.description == description_2
         self.assertWarns(UserWarning, get_module, **dict(nwbfile=nwbfile, name=name_1, description=description_2))
 
-    def test_make_nwbfile_from_metadata(self):
-        with self.assertRaisesWith(
-            exc_type=AssertionError,
-            exc_msg=(
-                "'session_start_time' was not found in metadata['NWBFile']! Please add the correct start time of the "
-                "session in ISO8601 format (%Y-%m-%dT%H:%M:%S) to this key of the metadata."
-            ),
-        ):
+    def test_make_nwbfile_from_metadata_empty(self):
+        with self.assertRaises(ValidationError):
             make_nwbfile_from_metadata(metadata=dict())
+
+    def test_make_nwbfile_from_metadata_session_start_time(self):
+        with self.assertRaises(ValidationError):
+            make_nwbfile_from_metadata(metadata=dict(NWBFile=dict(session_description="Mouse exploring an open field")))
```

Comment:
To reflect key differences that are now being made in this PR, can you add a test that ensures the `metadata` dict being passed is not modified in-place (i.e., the `session_description` placeholder and `uuid` are not actually set back to the dictionary)? And add a docstring note to that test explaining the context and desired behavior?